rplugin/python3/molten/code_cell.py

```diff
@@ -69,3 +69,11 @@ def get_text(self, nvim: Nvim) -> str:
             return "\n".join(
                 [lines[0][self.begin.colno :]] + lines[1:-1] + [lines[-1][: self.end.colno]]
             )
+
+    def calculate_output_offset(self) -> int:
+        lines = self.get_text(self.nvim).splitlines()
+        for i, line in enumerate(reversed(lines)):
+            # Show after first non-empty, non-comment line.
+            if line != '' and not line.startswith("#"):
```

Comment:
Molten supports a ton of languages, so I'd really like to stay away from a hard coded comment string. Making this configurable (with a list of possible comment strings), or using treesitter + the active kernel's language to get the correct comment string is better. Probably prefer the TS method.

Reply:
I think that just making this configurable would be fine, and a lot less effort

Reply:
Done.